.github/workflows/inspektor-gadget.yml

```diff
@@ -394,9 +374,12 @@ jobs:
         tar zxvf /home/runner/work/inspektor-gadget/kubectl-gadget-linux-amd64.tar.gz
         mv kubectl-gadget kubectl-gadget-linux-amd64
 
+        export CONTAINER_REPO="${{ env.REGISTRY }}/${{ env.CONTAINER_IMAGE }}"
+
         TESTS_DOCKER_ARGS="-e KUBECONFIG=/root/.kube/config -v /home/runner/.kube:/root/.kube -v /home/runner/work/_temp/.minikube:/home/runner/work/_temp/.minikube" \
             make -C integration build test
 
+        sed -i "s@ghcr.io/kinvolk/inspektor-gadget@${CONTAINER_REPO}@g" integration/gadget-integration-tests-job.yaml
```

Comment:
Mmmm I am not sure I understood why it is necessary. We only push this image for releases so `CONTAINER_REPO` will always be `ghcr.io/kinvolk/inspektor-gadget`, isn't it?

Reply:
This replace is done for fork if they want to release their own test assets somewhere without having kinvolk name inside them.